policyengine_us/parameters/gov/states/ky/tax/income/credits/non_refundable.yaml

```diff
@@ -6,4 +6,3 @@ values:
   2021-01-01:
     - ky_family_size_tax_credit
     - ky_tuition_tax_credit
-    - ky_personal_tax_credits
```

Comment:
explain?

Reply:
Why not make `ky_personal_tax_credits` a new variable based on the indiv/joint choice and keep it in the list?

Reply:
Done!